### ISSUE TITLE
Схема

### ISSUE THREAD

**Issue by Flyguy** [Skiff2/SimplyVFD_V3#1]
Что то с схемой совсем беда....
24 нога с терморезистора, зачем то, подключена к +400в.
14 нога (бутстрепная земля), зачем то, подключена к микроконтроллеру...

**Comment by Eksol** [comment]
Согласен. В спринте есть какая то неизвестная микросхема на 16 ног. А в схеме ее нет совсем.

**Comment by Eksol** [comment]
Если это проект opensource то можно ли сюда выкладывать свои версии платы? 
Первое что мне не нравится, это прокладка между процессором и основной платой. Хотелось бы развести плату прямо под проц. вывести 4 пина под программатор. 
Второе что мне не нравится это модульность. По идее можно сделать версию платы ALL IN ONE. То есть все в одном. Конденсаторы, выпрямители, релюшки.

**Comment by Skiff2** [comment]
Да, версии платы можно. Только с пометкой в названии что это пользовательская. Ибо бывает что платы делают так, что частотник потом взрывается и мне бы не хотелось с этим разбираться.
